CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_hubspot v0.12.0
+- Removes the dependencies on [dbt-expectations](https://github.com/calogica/dbt-expectations/releases) and [dbt-date](https://github.com/calogica/dbt-date/releases). ([PR #118](https://github.com/fivetran/dbt_hubspot/pull/118))
```

Comment:
Can we include a sub header to this CHANGELOG entry

Reply:
Additionally, can we make a note about the tests that have been removed

Reply:
added